duo_client/https_wrapper.py

```diff
@@ -118,7 +118,8 @@ def connect(self):
                                              self.timeout)
         if self._tunnel_host:
             self._tunnel()
-        self.sock = self.default_ssl_context.wrap_socket(self.sock)
+        self.sock = self.default_ssl_context.wrap_socket(self.sock,
+                                                         server_hostname=self.host)
```

Comment:
Do we need to do any check on self.host?  that it's not None for instance?

Reply:
Line 63 says it can be "host:port" - would we need to strip off the port piece here?

Reply:
I just added a test case for for host:port form. Checking for None isn't necessary: the constructor excepts if host is None